sdk/monitor/monitor-opentelemetry-exporter/src/export/base.ts

```diff
@@ -93,11 +104,25 @@ export abstract class AzureMonitorBaseExporter {
    */
   protected async _exportEnvelopes(envelopes: Envelope[]): Promise<ExportResult> {
     diag.info(`Exporting ${envelopes.length} envelope(s)`);
+    
+    // Shutdown statsbeat if the maximum number of failures is exceeded
+    if (
+      this._statsbeatFailureCount > MAX_STATSBEAT_FAILURES &&
+      this._statsbeatMetrics &&
+      this._statsbeatMetrics.isEnabled()
+    ) {
+      this._statsbeatMetrics.enable(false);
```

Comment:
Disabling and then shutting down sound redundant, maybe just shut it down and remove enable/disable logic, add enable logic in constructor

Reply:
Makes sense, moved this logic into the constructor to simplify.